src/stores/spaces.ts

```diff
@@ -194,15 +216,16 @@ export const useSpacesStore = defineStore("spaces", () => {
     bulkSetBuildingOccupancy,
     setBuildingOccupancy,
     buildings,
+    setRooms,
     setSpaces,
-    bulkSetSpaceOccupancy,
-    setSpaceOccupancy,
+    bulkSetRoomOccupancy,
+    setRoomOccupancy,
     spaces,
     filteredSpaces,
     clearFilters,
     isFiltered,
-    getSpaceById,
-    buildingsI18n,
+    buildingsI18n, //These need to be exported to be passed as payload from server to client
```

Comment:
Not sure I understand this comment, is this only applicable for `buildingsI18n`?

Reply:
All 3 of `buildingsI18n`, `roomsI8n` and `spacesI18n` could have been removed from the `return` of the store since they are never directly used from the outside. However, if we do that, they stop being included in the store's payload that's pre-rendered on the server and serialised to be picked up by browser code. This is not documented and I thought it could be overlooked during some future refactoring. The overall process of Pinia hydration seems a bit messy :(